src/main/java/io/seqera/tower/cli/commands/runs/DumpCmd.java

```diff
@@ -167,15 +170,36 @@ private void dumpTasks(PrintWriter progress, TarArchiveOutputStream out, Long ws
     private void dumpWorkflowDetails(PrintWriter progress, TarArchiveOutputStream out, Long wspId) throws ApiException, IOException {
         progress.println(ansi("- Workflow details"));
 
+        // User info
+        UserDbDto user = api().userInfo().getUser();
+
+        // General workflow info
         DescribeWorkflowResponse workflowResponse = workflowById(wspId, id);
         Workflow workflow = workflowResponse.getWorkflow();
         if (workflow == null) {
             throw new TowerException("Unknown workflow");
         }
+
+        // Launch info
+        Launch launch = null;
+        Long pipelineId = null;
+        if (workflow.getLaunchId() != null) {
+
+            launch = launchById(wspId, workflow.getLaunchId());
+
+            DescribeWorkflowLaunchResponse wfLaunchResponse = workflowLaunchById(wspId, workflow.getId());
+            if (wfLaunchResponse != null && wfLaunchResponse.getLaunch() != null) {
+                pipelineId = wfLaunchResponse.getLaunch().getPipelineId();
+            }
+        }
+
+        // Load and metrics info
         WorkflowLoad workflowLoad = workflowLoadByWorkflowId(wspId, id);
-        Launch launch = workflow.getLaunchId() != null ? launchById(wspId, workflow.getLaunchId()) : null;
         List<WorkflowMetrics> metrics = api().describeWorkflowMetrics(workflow.getId(), wspId).getMetrics();
-        addEntry(out, "workflow.json", Workflow.class, workflow);
+
+        WorkflowDumpExportFormat wfDetails = new WorkflowDumpExportFormat(workflow, pipelineId, wspId, user.getId(), user.getEmail());
```

Comment:
For me to create a new class that extends `Workflow` class with it's special serializer it's a bit dangerous. At some point `Workflow` can add new attributes that clash with these attributes.
 
If it's a real requirement to add the attributes in the same `workflow.json` then seems ok to me. But most likely to put all this data into a new JSON (example: `workflow-info.json`) is also a valid solution for our users and the implementation is easier and a bit more safe. 

What do you think @ejseqera ?

Reply:
Not necessarily a real requirement to add the attribute in `workflow.json`, it can be elsewhere, as long as it is captured somewhere. Pulling metadata that pertains to the who/what/when/how into `workflow-info` wouldn't be bad at all.

Will pulling it in `workflow-info.json` as an example, make it easier to retrieve additional attributes in the future that we may not be capturing now?

Reply:
> Will pulling it in workflow-info.json as an example, make it easier to retrieve additional attributes in the future that we may not be capturing now?

Yes, that would make it a lot easier because that way we don't need to artificially add extra fields to the base entity (`Workflow` entity), both files can evolve separately without fear of overwriting each other fields

I will add the separate metadata file then